nucypher/cli/commands/alice.py

```diff
@@ -393,7 +384,7 @@ def grant(general_config,
         if any((value, rate)):
             raise click.BadOptionUsage(option_name="--value, --rate",
                                        message="Can't use --value or --rate with a federated Alice.")
-    elif not (bool(value) ^ bool(rate)):
+    if bool(value) and bool(rate):
```

Comment:
Why this change? Now it misses when `bool(value)==False` and `bool(rate)==False`